### PR TITLE
Reduce utilities size

### LINKED ISSUE #274: [DISCUSSION] Utilities.
### Context
There's an ongoing feeling on the Airship squad about utilities. For some of us there's no clear usage for them, for some others it's a great tool to have under our belt.

Main concerns:
- There are a lot of CSS only for them.
- Flex classes feels unnecessary.

Let's debate here and review the utilities generated CSS to address those concerns.

### PR DESCRIPTION
**Pull request by IagoLast**
Fix #274 
---

### CODE REVIEW

**Comment by jesusbotella** [comment]
I think the change is good but this topic needs an agreement between all parties 😂

**Comment by jesusbotella** [comment]
Have you checked the size difference between bundles?

File: packages/styles/src/utilities/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Where *sides* is one of:
 
 Where *size* is one of:
 
-`0`, `4`, `8`, `12`, `16`, `20`, `24`, `28`, `32`, `36`, `40`, `48`, `60`, `64`, `72`, `80`, `96`, `100`, `120`, `128`, `140`, `148`, `152`, `240`
+`4, 8, 16, 32, 64, 128`
```
Comment (ivanmalagon) [inline packages/styles/src/utilities/README.md]: Let's do it from 0 to 40.

**Comment by IagoLast** [comment]
Final utilities size: 6.05KB

**Pull request merged**